lib/ts/recipe/session/recipe.ts

```diff
@@ -114,7 +114,7 @@ export default class Session extends RecipeModule<unknown, unknown, unknown, any
     };
 
     private async getSessionContext({ action }: RecipeEvent): Promise<SessionContextType> {
-        if (action === "SESSION_CREATED" || action === "REFRESH_SESSION") {
+        if (action === "SESSION_CREATED" || action === "REFRESH_SESSION" || action === "GRANT_MISSING") {
```

Comment:
We are checking if the action type if `GRANT_MISSING`, but not doing anything specific for it in the if condition?

Reply:
The session context is the same in that case, we still have a userId and the payload.